src/Manager.php

```diff
@@ -65,12 +65,12 @@ public function repository()
     /**
      * Return Table
      *
-     * @return \Cake\ORM\Table Table Instance
+     * @return \Cake\Datasource\RepositoryInterface Repository Instance
```

Comment:
It fixes only the doc block (semantic) to what the non-deprecated method already returns.